docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -93,6 +93,34 @@ catalog named ``sales`` using the configured connector.
 Type mapping
 ------------
 
+The data type mappings are as follows:
+
+=================== ================================ =======================================================================
+PostgreSQL          Trino                            Notes
+=================== ================================ =======================================================================
+``BIT``             ``BOOLEAN``
+``SMALLINT``        ``SMALLINT``
+``INTEGER``         ``INTEGER``
+``BIGINT``          ``BIGINT``
+``REAL``            ``DATE``
+``DOUBLE``          ``DOUBLE``
+``NUMERIC(p, s)``   ``DECIMAL(p, s)``                See :ref:`postgresql-decimal-type-handling` for more information.
+``CHAR(n)``         ``CHAR(n)``
+``VARCHAR(n)``      ``VARCHAR(n)``
+``BINARY``          ``VARBINARY``
+``DATE``            ``DATE``
+``TIME``            ``TIME``
+``TIMESTAMP``       ``TIMESTAMP``
+``TIMESTAMPTZ(n)``  ``TIMESTAMP(n) WITH TIME ZONE``
+``MONEY``           ``VARCHAR``
+``UUID``            ``UUID``
+``JSONB``           ``JSON``                         ``JSONB`` is an alias of ``JSON``.
```

Comment:
JSONB and JSON are different types in PostgreSQL. We do handle both though by mapping to Trino JSON.